mon-pix/tests/integration/components/feedback-certification-section-test.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupRenderingTest } from 'ember-mocha';
+import { render, find } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+describe('Integration | Component | feedback-certification-section', function() {
+  setupRenderingTest();
+
+  it('renders', async function() {
+
+    await render(hbs`{{feedback-certification-section}}`);
+
+    expect(find('.feedback-certification-section__div').textContent.trim()).to.contain('Pour signaler un problème, appelez votre surveillant et communiquez-lui les informations suivantes :');
```

Comment:
Côté team prescription on a réalisé un helper appelé [contains](https://github.com/1024pix/pix/blob/dev/mon-pix/tests/helpers/contains.js) (il est présent dans PixApp et PixOrga). N'ayant pas besoin de sélecteur CSS il permet de s'affranchir de l'implémentation du composant pour vérifier que celui-ci fonctionne.

Il suffit de l'importer `import { contains } from '../../helpers/contains';` et l'utiliser comme suit :

```suggestion
    expect(contains('Pour signaler un problème, appelez votre surveillant et communiquez-lui les informations suivantes :')).to.exist;
```

Reply:
@frinyvonnick Merci encore pour ton aide, pour l'instant, comme convenu tous ensemble ce matin, nous le laissons tel quel :)